src/components/Taf/Taf.jsx

```diff
@@ -695,6 +638,7 @@ class Taf extends Component {
       case READ_ABILITIES.COPY['dataField']:
         return copiedTafRef === selectedTaf.tafData.metadata.uuid;
       case READ_ABILITIES.CORRECT['dataField']:
+        return (tafType === LIFECYCLE_STAGE_NAMES.CANCEL);
```

Comment:
From documentation I understood that currently only a TAF inside valid period can be corrected.